tests/connections.py

```diff
@@ -191,7 +190,7 @@ def create_connections_until_exhaustion(
                         client.post(
                             "/log/private",
                             {"id": 42, "msg": "foo"},
-                            timeout=3 if IS_SNP else 1,
+                            timeout=3 if args.enclave_platform == "snp" else 1,
```

Comment:
Unsure if we still need SNP-specific timeouts? But plausible, so leaving them for now.